io/private/las/Vlr.hpp

```diff
@@ -57,6 +57,8 @@ static const uint16_t TextDescriptionRecordId = 3;
 static const uint16_t ExtraBytesRecordId = 4;
 static const uint16_t PdalMetadataRecordId = 12;
 static const uint16_t PdalPipelineRecordId = 13;
+static const uint16_t PdalWkt2RecordId = 4224;
```

Comment:
As referenced in the ticket, we changed this to LASFWkt2recordId. Maybe we should change the code to reflect that too.

Reply:
done